symengine/tests/basic/test_subs.cpp

```diff
@@ -152,6 +152,11 @@ TEST_CASE("Mul: subs", "[subs]")
     r1 = mul(x, y)->subs({{x, real_double(0.0)}});
     r2 = real_double(0.0);
     REQUIRE(eq(*r1, *r2));
+
+    d.clear();
+    r1 = div(one, x);
```

Comment:
@isuruf  in 1/(x*y) case should i add a test with x as 0 and y as 0 [gives output ComplexInf] or only x as 0[gives output ComplexInf/y] ?